crates/pathfinder/src/rpc/pathfinder.rs

```diff
@@ -0,0 +1,16 @@
+pub fn register_all_methods(module: &mut jsonrpsee::RpcModule<()>) -> anyhow::Result<()> {
+    use anyhow::Context;
+
+    module
+        .register_async_method(
```

Comment:
You can also use a sync version if don't want to create a future for such a simple handler:
[`register_method`](https://docs.rs/jsonrpsee-http-server/0.11.0/jsonrpsee_http_server/struct.RpcModule.html#method.register_method)